core/server/models/settings.js

```diff
@@ -219,14 +220,36 @@ Settings = ghostBookshelf.Model.extend({
         });
     },
 
-    populateDefaults: function populateDefaults(unfilteredOptions) {
+    populateDefaults: async function populateDefaults(unfilteredOptions) {
         const options = this.filterOptions(unfilteredOptions, 'populateDefaults');
         const self = this;
 
         if (!options.context) {
             options.context = internalContext.context;
         }
 
+        // this is required for sqlite to pick up the columns after db init
+        await ghostBookshelf.knex.destroy();
+        await ghostBookshelf.knex.initialize();
```

Comment:
I don't know exactly why this is necessary but without it the columnInfo query below returns an empty result set.

Reply:
:disappointed: sucks for now, would really like to spend some time to find out why though